test/over_react_test/custom_matchers_test.dart

```diff
@@ -453,8 +455,11 @@ main() {
             'random log',
             'pizza',
             'nonsense',
-            'Failed prop type: combination error',
           ];
+
+          if (runtimeSupportsPropTypeWarnings()) {
+            logs = [...logs, 'Failed prop type: combination error'];
+          }
```

Comment:
Does this need to change? In the test that uses this error, the error being a prop error is irrelevant as the matcher is just looking for a specific log. It could definitely be clearer by making this string something else, but I'd expect the tests to pass in both DDC and dart2js as is

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f